python/cudf/cudf/core/dataframe.py

```diff
@@ -1163,7 +1164,10 @@ def _clean_nulls_from_dataframe(self, df):
         filling with `<NA>` values.
         """
         for col in df._data:
-            if self._data[col].has_nulls:
+            if is_list_dtype(self._data[col]):
+                # TODO we need to handle this
+                pass
```

Comment:
This makes me very weary, any other options, i.e. throw if there's nulls in the ListColumn for now?

Reply:
The net effect here is that `None` is used rather than `<NA>` to represent nulls for list columns for now. I think that's fine? If we threw here, we aren't able to `__repr__` list columns..